src/Calculator.jsx

```diff
@@ -143,47 +166,129 @@ const Numbers = styled.section`
   }
 `;
 
-const Calculator = () => (
-  <Container>
-    <SubTitle>
-      Please select your daily activity and choose the amout by clicking on it again
-    </SubTitle>
-    <Activitys>
-      <IconContainer>
-        <Icon>
-          <Img src="/bottle.png" alt="bottle" />
-        </Icon>
-        <Icon>
-          <Img src="/glass.png" alt="glass" />
-        </Icon>
-        <Icon>
-          <Img src="/flush.png" alt="flush" />
-        </Icon>
-      </IconContainer>
-      <IconContainer>
-        <Icon>
-          <Img src="/washing-hands.png" alt="washing hands" />
-        </Icon>
-        <Icon>
-          <Img src="/shower.png" alt="shower" />
-        </Icon>
-        <Icon>
-          <Img src="/plumbing-pipe.png" alt="plumbing pipe" />
-        </Icon>
-      </IconContainer>
-      <Calculate>CALCULATE</Calculate>
-    </Activitys>
-    <ResultsContainer>
-      <Results>
-        <ResultsTitle>Your cost</ResultsTitle>
-        <Numbers>5</Numbers>
-      </Results>
-      <Results>
-        <ResultsTitle>Liters</ResultsTitle>
-        <Numbers>5</Numbers>
-      </Results>
-    </ResultsContainer>
-  </Container>
-);
+// Functionality
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userChoices: {
+        Toilet: 0,
+        Shower: 0,
+        'Faucet tap': 0,
+        'Washing hands': 0,
+        Cup: 0,
+        Bottle: 0,
+      },
+      cost: 0,
+      liters: 0,
+      shows: {
+        Toilet: 0,
```

Comment:
i would say all the keys in shows change the to booleans ( true and false

Reply:
I tried using it as true and false
it wont work..

the 'show' keys are the opacity values in the style.component literally 

apperantly opacity: true and opacity: false doesnt work